### PR TITLE
Minimalist React implementation

### PR DESCRIPTION
**Pull request by jgimbel**
No webpack, browserify, babel, or any build tools, just pure in-browser tools that come to 2KB of HTML/JS.

### CODE REVIEW

**Comment by barberboy** [comment]
Sorry for closing the PR. I changed the default branch to be `gh-pages` and deleted `master`.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<html>
+<head>
+	<title>This converts temperatures</title>
+</head>
+<body>
+	<script src="https://fb.me/react-0.13.0.min.js" type="text/javascript"></script>
```
Comment (crodeheaver) [inline index.html]: Aren't you a version or two behind?

Comment (jgimbel) [reply index.html]: Yep! Back when I could have a jsx transformer in the browser.

File: temperature.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    return (
+      <main>
+        <form onSubmit={this.calculate}>
+          <label>
+            Tempature
+            <input type='number' name='temp'></input>
+          </label>
+          <label>
+            Farenheit
+            <input type='radio' name='type' value='toCelsius'/>
```
Comment (barberboy) [inline temperature.js]: I'd recommend setting one of these as the default.

**Comment by NathanBland** [comment]
This code is all fine and good.. where is the `gh-pages` link so I can see this actually do something?

**Comment by jgimbel** [comment]
[gh-pages](https://jgimbel.github.io/temperature-converter/) for @NathanBland

File: temperature.js (hunk 2 above)
Comment (jgimbel) [reply temperature.js]: Good point. Fixed in 644034f

File: index.html (hunk 1 above)
Comment (crodeheaver) [reply index.html]: Do they not allow that anymore? Too many people abuse it or something? haha

Comment (jgimbel) [reply index.html]: Kinda? It was never meant to be used in production. It was deprecated right when I started learning. I believe the last straw was when babel included jsx transform by default. [Docs about deprecation](https://facebook.github.io/react/blog/2015/06/12/deprecating-jstransform-and-react-tools.html)

**Comment by barberboy** [comment]
Looks good, Joel! 👍

**Pull request closed without merging**